pegasus/sites.v3/hourofcode.com/views/theme_common_head_after.haml

```diff
@@ -1,3 +1,5 @@
+=view :theme_google_analytics
```

Comment:
I figured there was a chance someone could add a call to `experiment.isEnabled` into `theme_common_header_after.js` in the future, which would depend on `window.trackEvent` existing.